src/TimeSlotGroup.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { PropTypes, Component } from 'react'
+import TimeSlot from './TimeSlot.jsx'
+import date from './utils/dates.js'
+import localizer from './localizer'
+
+export default class TimeSlotGroup extends Component {
+  static propTypes = {
+    timeslots: PropTypes.number.isRequired,
+    step: PropTypes.number.isRequired,
+    value: PropTypes.instanceOf(Date).isRequired,
+    showLabels: PropTypes.bool,
+    isNow: PropTypes.bool,
+    timeGutterFormat: PropTypes.string,
+    culture: PropTypes.string
+  }
+  static defaultProps = {
+    timeslots: 2,
+    step: 30,
```

Comment:
my explanation above for why the redundant required/default value is present hopefully covers this?